std/typecons.d

```diff
@@ -759,31 +759,79 @@ template Tuple(Specs...)
             return h;
         }
 
-        void toString(DG)(scope DG sink)
+        /**
+            Formats `Tuple` with either `%s`, `%(inner%)` or `%(inner%|sep%)`.
+
+            `%s` is the original format.
+            `%(inner%)` where `inner` is the format applied the expanded `Tuple`,
+            so `inner` may contain as many formats as the `Tuple` has fields.
+            `%(inner%|sep%)`, where `inner` is one format, that is applied
+            on all fields of the `Tuple`. The format must be compatible to all
+            of them.
+        */
+        //void toString(DG, Char)(scope DG sink, std.format.FormatSpec!Char fmt = "%s") const
+        void toString(DG)(scope DG sink, std.format.FormatSpec!char fmt = "%s") const
```

Comment:
Function templates can be overloaded with non-templated functions now, so the other overload can lose a pair of parantheses.